### DISCUSSION TITLE
Release 2.37.1 missing Bionic amd64?

### DISCUSSION THREAD

**Discussion by m1lkman** [trapexit/mergerfs#1252] [General]
is there no Ubuntu Bionic amd64 deb file for 2.37.1? I don't see it listed on the release page.

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs/releases/download/2.37.1/mergerfs_2.37.1.ubuntu-bionic_amd64.deb

**Comment by trapexit** [comment]
Sometimes builds fail and I don't have a good error handling setup yet. It takes several hours to build everything so things slip through.

**Reply by m1lkman** [reply]
Thank you
